packages/math-input/src/components/keypad/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,71 @@
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import * as React from "react";
+import "@testing-library/jest-dom";
+
+import Button from "../button";
+
+describe("<Button />", () => {
+    it("uses the aria label", () => {
+        // Arrange
+        render(
+            <Button onPress={jest.fn()} ariaLabel="Oranges">
```

Comment:
```suggestion
            <Button onPress={() => {}} ariaLabel="Oranges">
```

I don't think we need `jest.fn` here since we're not spying on anything